src/ts/count-characters.ts

```diff
@@ -1,12 +1,15 @@
 import type {Action} from './types';
 
-export const countCharacters: Action<HTMLInputElement | HTMLTextAreaElement> = node => {
+export const countCharacters: Action<HTMLInputElement | HTMLTextAreaElement> = (
+  node,
+  countCharactersLeftLabel: string = 'tegn igjen'
```

Comment:
Since we can only have one parameter as input to an action, I think it would be most future proof to let it be an object, with the value inside it. That way it can be expanded with more parameters later.

This same principle goes for custom events too.

```suggestion
  parameters: { countCharactersLeftLabel: string } = {countCharactersLeftLabel: 'tegn igjen'}
```

Reply:
Agree. Changing it now.